index.html

```diff
@@ -0,0 +1,111 @@
+                    </ul>
+            </nav>
+        </header>
+        <main class="container">
+            <h1 class="visually-hidden">Барбершоп Бородинский" </h1>
+            <section class="features">
+                <h2 class="visually-hidden">Преимущества</h2>
+                    <ul class="features-list">
+                        <li class="feature-item">
+                            <h3>Быстро</h3>
+                            <p>Мы делаем свою работу быстро! Два часа пролетят незаметно и вы - счастливый обладатель стильной стрижки-минутки!</p>
+                        </li>
+                        <li class="feature-item">
+                            <h3>Круто</h3>
+                            <p>Забудь, как вы стриглись раньше. <br> Мы сделаем из вас звезду футбола или кино!<br>Во всяком случае внешне.</p>
+                        </li>
+                        <li class="feature-item">
+                            <h3>Дорого</h3>
+                            <p>Наши мастера - профессионалы своего дела и не могут стоить дешево. К тому же разве цена не дает опреленный статус?</p>
+                        </li>
+
+                    </ul>
+
+
+            </section>
+            <section class="news">
+                <h2>Новости</h2>
+                <ul class="news-list">
+                    <li class="news-item">
+                    <p>Нам наконец завезли ягермейстер!Теперь вы моежете пропустить стаканчик во время стрижки</p>
+                    <time datatime="2016-01-11">11января</time>
+                    </li>
+                    <li class="news-item">
+                    <p>В нашей команде пополнение, Борис "Бритва" Стригунец, обладатель множества титулов наград пополнил наши стройные ряды</p>
+                    <time datetime="2016-01-18">18 января</time>
+                    </li>
+                </ul>
+                <a class="button" href="news.html">Все новости</a>
+            </section>
+            <section class="gallary"> 
+                <h2>Фотогалерея</h2>
+                <figure class="gallary-content">
+                    <a href="#"><img src="img/studio.jpg" width="286 height=164" alt="Интерьер"></a>
+                </figure>
+                <span class="button gallary-button gallary-button-back">Назад</span>
+                <span class="button gallary-button gallary-button-next">Вперед</span>
+            </section>
+            <section class="contacts">
+                <h2>Контактная информация</h2>
+                <p>
+                    Барбершоп "Бородинский"<br>
+                    Адрес: г. Санкт-Петербург, Б. Конюшенная, д. 19/8<br>
+                    Телефон: +7 (812) 666-02-66
+                </p>
+                <p>
+                    Время работы:<br>
+                    пн-пт: с 10:00 до 22:00<br>
+                    сб-вс: с 10:00 до 19:00
+                </p>
+            <a class="button" hraf="map.html">Как проехать</a>
```

Comment:
```suggestion
            <a class="button" href="map.html">Как проехать</a>
```

+ отформатируй их нормально, у тебя они на уровне с родителем